src/compiler.js

```diff
@@ -511,7 +511,7 @@ var Compiler = Object.extend({
             var name = target.value;
             var id = frame.lookup(name);
 
-            if (id == null) {
+            if (id === null || id === undefined) {
```

Comment:
The practice leads us to always use `===` that `==` will confuse other what is exactly equal to.

But it works fine in this case.